backend/plugins/frontline_api/src/modules/reports/graphql/reportQueries.ts

```
@@ -34,13 +34,25 @@ export const reportQueries = {
         },
       },
 
+      {
+        $addFields: {
+          createdAtDate: {
+            $cond: {
+              if: { $eq: [{ $type: '$createdAt' }, 'string'] },
+              then: { $dateFromString: { dateString: '$createdAt' } },
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)